src/functionBased/displays/About.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { NavLink, Route, Routes } from 'react-router-dom';
+// import { Switch } from 'react-router-dom/cjs/react-router-dom.min';
```

Comment:
I saw that you already used Routes instead of Switch from react-router-dom this line will not be required.